e2e/helm.e2e.spec.ts

```diff
@@ -85,10 +85,10 @@ test.describe.serial('Helm Deployment Test', () => {
     // Get Node Port number.
     const nodePortNumber = (await kubectl('get', '--namespace', 'default', '--output=jsonpath={.spec.ports[0].nodePort}', 'services', 'nginx-sample')).trim();
 
-    const podName = (await kubectl('get', 'pods', '--output=name', '--namespace', 'default')).trim();
+    const podName = (await kubectl('get', 'pods', '--output=name', '--namespace', 'default')).split(/\s+/).filter(pod => pod.includes('pod/nginx-sample'))[0].trim();
```

Comment:
```suggestion
    const podName = (await kubectl('get', 'pods', '--output=name', '--namespace', 'default')).split(/\s+/).find(pod => pod.includes('pod/nginx-sample')).trim();
```
This line is also somewhat long; consider using intermediate variables.